### DISCUSSION TITLE
Advanced support for <kbd> tag

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#17926] [Other Feature Feedback, Questions, & Ideas]
Something a few people may not know about is, that you can use the `<kbd>` HTML tag in markdown to render keyboard-keys (or at least a somewhat similar style to them).

I like to use them for instructions such as "Press <kbd>Ctrl</kbd>+<kbd>C</kbd> to copy the content" in my markdown files but also other places such as the Issue forms.

However, I feel like GitHub could add a bit more support to this HTML tag.
When looking at other Markdown-renderers, or software that hooks into and enhances those, can you often see special support for specific keys.
As an example does `ctrl` have the `^` symbol added or `cmd` has the `⌘` symbol.

It could help improve readmes visually by making special keys more easily noticeable using their unique symbols.

Example from docs I make:
![image](https://user-images.githubusercontent.com/11576465/172032132-ac22ad67-2d50-4898-9dca-bdaf02c39861.png)

The implementation could also be relatively simple by just applying a `key-<name>` class to the `<kbd>` object where `<name>` would be whatever the content of the tag is.

As an example, would...
```markdown
<kbd>Ctrl</kbd>+<kbd>C</kbd>
```
...render as...
```html
<kbd class="key-ctrl">Ctrl</kbd>+<kbd class="key-c">C</kbd>
```

This could then allow GitHub to apply symbols using the `:before` attribute in CSS for those special keys.

Another thing that should perhaps be considered is maybe a markdown-friendly syntax to indicate keys to be rendered.
The above shown image uses `++ctrl+c++ / ++cmd+c++` as format to indicate those special types.
Whether this would fit with Markdown's semantics (or if MD not even has a format to begin with) is up for discussion.

Those are some relatively minor requests but I see some good potential here, so I hope they will be considered.

**Comment by queenofcorgis** [comment]
Thanks for leaving a request and the level of detail! I labeled this so the appropriate team can easily find and review your request :octocat:
